README.md

```diff
@@ -24,14 +24,15 @@ For non Sublime development a bunch of other IDE plugins are available [here](ht
 ## Log style
 To try and keep consistency across our systems
 type and namespace
-query    -> payload
-reply    -> payload
-request  -> payload
-response -> payload
-log      -> system
-responseTime -> system
-system   -> error
-thirdPartySystem -> error
+
+		query    -> payload
```

Comment:
Yeah I agree looks better formatted, might be better just using back tickets a la -

```
reply    -> payload
request  -> payload
response -> payload
log      -> system
responseTime -> system
system   -> error
thirdPartySystem -> error
```